src/java/picard/vcf/filter/FilterVcf.java

```diff
@@ -20,6 +20,7 @@
  * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
+ * 
  */
 package picard.vcf.filter;
 
```

Comment:
My sense is that this is the wrong file for this. This file modifies VCFs FILTER field, not what you want, right? I'm assuming you want to actually remove records from the VCF. Please correct me if I'm wrong about this.